examples/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
+PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
+PythonPlot = "274fc56d-3b97-40fa-a1cd-1b4a50311bf9"
+ScikitLearn = "3646fa90-6ef7-5e7e-9f22-8aca16db6324"
```

Comment:
If /examples are for users then I suggest including a Manifest.toml as well, for reproducibility.

Reply:
Done!